worldedit-cli/src/main/java/com/sk89q/worldedit/cli/CLIPlatform.java

```diff
@@ -153,6 +156,11 @@ public String getPlatformVersion() {
         return capabilities;
     }
 
+    @Override
+    public Collection<SideEffect> getSupportedSideEffects() {
+        return Lists.newArrayList();
```

Comment:
Probably doesn't matter, but this can use `ImmutableList.of()`, `Collections.emptyList()`, or any other constant empty collection to avoid a tiny allocation. It's generally more performant and compact as well.